content/en/docs/community/release_checklist.md

```diff
@@ -4,44 +4,67 @@ description: "Checklist for maintainers when releasing the next version of Helm.
 weight: 2
 ---
 
-**IMPORTANT**: If your experience deviates from this document, please document
-the changes to keep it up-to-date.
+# A Maintainer's Guide to Releasing Helm
 
-## A Maintainer's Guide to Releasing Helm
-
-So you're in charge of a new release for helm? Cool. Here's what to do...
-
-![TODO: Nothing](../../images/nothing.png)
-
-Just kidding! :trollface:
+Time for a new Helm release! As a Helm maintainer cutting a release, you are
+the best person to [update this
+release checklist](https://github.com/helm/helm-www/blob/master/content/en/docs/community/release_checklist.md)
+should your experiences vary from what's documented here.
 
 All releases will be of the form vX.Y.Z where X is the major version number, Y
 is the minor version number and Z is the patch release number. This project
 strictly follows [semantic versioning](https://semver.org/) so following this
 step is critical.
 
+These directions will cover initial configuration followed by the release
+process for three different kinds of releases:
+
+* Major Releases - released less frequently - may have breaking changes
+* Minor Releases - released regularly - no breaking changes
+* Patch Releases - released as needed - do not require all steps in this guide
+
+[Initial Configuration](#initial-configuration)
+
+1. [Create the Release Branch](#1-create-the-release-branch)
+2. [Major/Minor releases: Change the Version Number in Git](#2-majorminor-releases-change-the-version-number-in-git)
+3. [Major/Minor releases: Commit and Push the Release Branch](#3-majorminor-releases-commit-and-push-the-release-branch)
+4. [Major/Minor releases: Create a Release Candidate](#4-majorminor-releases-create-a-release-candidate)
+5. [Major/Minor releases: Iterate on Successive Release Candidates](#5-majorminor-releases-iterate-on-successive-release-candidates)
+6. [Major/Minor releases: Finalize the Release](#6-majorminor-releases-finalize-the-release)
+7. [PGP Sign the downloads](#7-pgp-sign-the-downloads)
+8. [Write the Release Notes](#8-write-the-release-notes)
```

Comment:
should this swap places with step 7? From our pairing session, it was quicker to write the release notes while waiting for the release to finalize, sign the downloads, then publish the release notes.

Reply:
Yes, I think you're right. I'm revising this in my next update.